x/staking/src/types/tx_proto.rs

```diff
@@ -72,6 +72,31 @@ pub struct Commission {
     update_time: Timestamp,
 }
 
+impl TryFrom<CommissionRaw> for Commission {
+    type Error = Error;
+
+    fn try_from(value: CommissionRaw) -> Result<Self, Self::Error> {
+        Ok(Self {
+            commission_rates: value
+                .commission_rates
+                .ok_or(Error::MissingField(
+                    "Value should exists. It's the proto3 rule to have Option<T> instead of T"
```

Comment:
I'm not sure a user would understand this message. Something like "missing field X" might be better?

Reply:
Thank you, I'll update the messages.